test/e2e/network/loadbalancer.go

```diff
@@ -926,7 +926,7 @@ var _ = common.SIGDescribe("LoadBalancers", feature.LoadBalancer, func() {
 	})
 })
 
-var _ = common.SIGDescribe("LoadBalancers ESIPP", feature.LoadBalancer, framework.WithSlow(), func() {
+var _ = common.SIGDescribe("LoadBalancers ExternalTrafficPolicy: Local", feature.LoadBalancer, framework.WithSlow(), func() {
```

Comment:
(FTR I'm doing this now just to keep all (most?) of the renaming in one PR; I didn't want people updating their CI configs to skip all the "ESIPP" tests and then have us rename them later...)